libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1154,5 +1159,80 @@
         "value": "This attribute does not have any value.",
         "comment": "",
         "limit": 0
+    },
+    "shareDialog.share.grantee.title": {
+        "value": "Share with users and groups",
+        "comment": "Title of share dialog list of shared grantee",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.share": {
+        "value": "Share",
+        "comment": "Title of confirm button of share dialog",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.list.title": {
+        "value": "Shared with",
+        "comment": "Title of shared list of grantees",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.add": {
+        "value": "Add",
+        "comment": "Title of add grantee button",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.add.empty.selection": {
+        "value": "No user or group selected.",
+        "comment": "Info that no user or group selected in grantee list",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.add.info": {
+        "value": "Add users and groups",
+        "comment": "Description of add grantee button or dialog",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.add.search.placeholder": {
+        "value": "Search users and groups",
+        "comment": "Information in search user and groups filed",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.add.search.no.matching.items": {
+        "value": "No matching users or groups.",
+        "comment": "Information that search query in input no match user and groups",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.item.you": {
+        "value": "You",
+        "comment": "Description that grantee is current user",
+        "limit": 0
+    },
+    "shareDialog.share.grantee.item.users": {
+        "value": "users",
```

Comment:
Please change it to `"{granteeCount} users"`.

Reply:
Done